python/orca/tutorial/tensorflow/ncf_orca.py

```diff
@@ -0,0 +1,155 @@
+
+    with tf.name_scope("GMF"):
+        user_embed_GMF = tf.keras.layers.Embedding(max_user_id + 1, embedding_size)(user)
+        item_embed_GMF = tf.keras.layers.Embedding(max_item_id + 1, embedding_size)(item)
+        GMF = tf.keras.layers.Multiply()([user_embed_GMF, item_embed_GMF])
+
+    with tf.name_scope("MLP"):
+        user_embed_MLP = tf.keras.layers.Embedding(max_user_id + 1, embedding_size)(user)
+        item_embed_MLP = tf.keras.layers.Embedding(max_item_id + 1, embedding_size)(item)
+        interaction = tf.concat([user_embed_MLP, item_embed_MLP], axis=-1)
+        layer1_MLP = tf.keras.layers.Dense(units=embedding_size * 2, activation='relu')(interaction)
+        layer1_MLP = tf.keras.layers.Dropout(rate=0.2)(layer1_MLP)
+        layer2_MLP = tf.keras.layers.Dense(units=embedding_size, activation='relu')(layer1_MLP)
+        layer2_MLP = tf.keras.layers.Dropout(rate=0.2)(layer2_MLP)
+        layer3_MLP = tf.keras.layers.Dense(units=embedding_size // 2, activation='relu')(layer2_MLP)
+        layer3_MLP = tf.keras.layers.Dropout(rate=0.2)(layer3_MLP)
+
+    # Concate the two parts together
+    with tf.name_scope("concatenation"):
+        concatenation = tf.concat([GMF, layer3_MLP], axis=-1)
+        outputs = tf.keras.layers.Dense(1, activation='sigmoid')(concatenation)
+
+    model = tf.keras.Model(inputs=[user, item], outputs=outputs)
+    model.compile(optimizer= "adam",
+                  loss= "binary_crossentropy",
+                  metrics=['accuracy'])
+    return model
+
+cluster_mode = str(sys.argv[1])
+# cluster_mode = "k8s"
+if cluster_mode == "local":
+    sc = init_orca_context(memory=str(sys.argv[2]))
+elif cluster_mode == "standalone":
+    sc = init_orca_context("standalone", cores=int(sys.argv[3]), num_nodes=int(sys.argv[4]), memory=str(sys.argv[2]))
+elif cluster_mode == "yarn":
+    sc = init_orca_context("yarn", cores=int(sys.argv[3]), num_nodes=int(sys.argv[4]), memory=str(sys.argv[2]))
+elif cluster_mode == "k8s":
+    sc = init_orca_context(cluster_mode="k8s", cores=8, num_nodes=4, memory="20g",
+                           master="k8s://https://172.16.0.200:6443",
+                           container_image="10.239.45.10/arda/intelanalytics/bigdl-k8s-spark-3.1.2:0.14.0-SNAPSHOT",
+                           conf={"spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName": "nfsvolumeclaim",
+                                 "spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl2.0/data",
+                                 "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName": "nfsvolumeclaim",
+                                 "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl2.0/data"})
+elif cluster_mode == "spark-submit":  # To test k8s using spark-submit
+    sc = init_orca_context(cluster_mode="spark-submit")
+
+
+data_path = "./bigdl2.0/data"
```

Comment:
change this

Reply:
> change this

What path should we read data from?

Reply:
Just the currently directory? To align with the PyTorch example?